po/es.po

```diff
@@ -425,9 +425,8 @@ msgid ""
 msgstr ""
 "<p>Esta versión amplía las funciones disponibles actualmente.</p><p> Características:"
 "</p>\n"
-"<ul><li> El nombre del color, especificado por las palabras clave de color w3c y la encuesta de color xkcd "
-", puede mostrarse ahora</li><li>Cambie el color introduciendo un color "
-", palabra clave</li><li>Un nuevo cuadro de diálogo de la paleta que muestra diferentes paletas de colores</"
+"<ul><li> El nombre del color, especificado por las palabras clave de color w3c y la encuesta de color xkcd"
+",puede mostrarse ahora</li><li>Cambie el color introduciendo un color, palabra clave</li><li>Un nuevo cuadro de diálogo de la paleta que muestra diferentes paletas de colores</"
```

Comment:
I think you deleted one space too many. Now there would be no space (`xkcd,puede`)